src/app/App.js

```diff
@@ -11,22 +12,32 @@ import {
   doc,
 } from "firebase/firestore";
 import { fetchTodos } from "../store/todos/actions";
+import {
+  fetchUsers,
+  updateUser as updateUserAction,
+  createUser as createUserAction,
+} from "../store/users/actions";
+import { selectUsers } from "../store/users/selectors";
+import { Button, Input } from "antd";
 
 function App() {
   const dispatch = useDispatch();
-  const [newName, setNewName] = useState("");
-  const [newAge, setNewAge] = useState(0);
-  const [users, setUsers] = useState([]);
+  const users = useSelector(selectUsers);
+
+  // const usersExample = useSelector((state) => state.users.users)
+
   const usersCollectionRef = collection(db, "users");
 
-  const createUser = async () => {
-    await addDoc(usersCollectionRef, { name: newName, age: Number(newAge) });
+  const createUser = async (name, age) => {
+    // await addDoc(usersCollectionRef, { name: newName, age: Number(newAge) });
+    // console.log("create User is hit");
+    dispatch(
+      createUserAction({ user: { name: name, age: Number(age) } })
```

Comment:
Your `createUserAction` looks like its parameter is an object with `name` and `age` so sending it like this with `user` will make it so those properties aren't seen